### DISCUSSION TITLE
Clarification of browser compatibility differences to SAPUI5

### DISCUSSION THREAD

**Discussion by tzimmermann** [UI5/webcomponents#9408] [General]
Hi all,

Looking at the [ui5-webcomponent's browser support table](https://github.com/SAP/ui5-webcomponents?tab=readme-ov-file#browser-support), it mentions "latest 2 stable releases" for all major browsers.

Comparing it with the [SAPUI5 browser compatibility table](https://help.sap.com/docs/SAP_NETWEAVER_AS_ABAP_751_IP/468a97775123488ab3345a0c48cadd8f/74b59efa0eef48988d3b716bd0ecc933.html#browser-and-platform-support-matrix), I'd like to clarify the following questions:

- Are you supporting Firefox ESR version ([currently 12 versions behind the latest](https://endoflife.date/firefox))?
- Do you support the same OS/device combinations as SAPUI5, or are there any limitations?

In case there are differences in compatibility between UI5-webcomponents and SAPUI5, I think it would be helpful to mention these in the README explicitly.

Thank you,
Thomas

**Comment by ilhan007** [comment]
Hello @tzimmermann 

Firefox ESR is not supported yet, we will start supporting Firefox ESR with its next release, scheduled on 9th July based on this [calendar](https://whattrainisitnow.com/calendar/) as we are using a feature (the Popover API, that is not available with the current latest ESR version and will be available with the next coming on 9th July).

We support the latest two releases of Safari and Chrome on MacOS and Edge, Chrome, Firefox on Windows
But I see the point, we did not document the OS/device relation and version, we definitely need to do that and update you once ready.


 BR,
 ilhan

**Reply by Aurelien-sig** [reply]
@ilhan007 Thanks for the feedback

Just quick point for clarification : 

**1 - There will be a support for the latest two releases on**
  - macOS: Safari and Chrome
     - but Firefox will be added?
   - Windows: Edge, Chrome, Firefox 
 
**2 - Enterprise specifics :**
   - Firefox ESR will be supported from tomorrow, but on what platforms ? 

Thanks

**Reply by Aurelien-sig** [reply]
@ilhan007 any feedback/update?

Thanks

FYI @tzimmermann
